admin/class-insert-rule-data.php

```diff
@@ -0,0 +1,119 @@
+<?php
+/**
+ * Inserts rule data about a post to the database
+ *
+ * @since 2.0.0
+ *
+ * @package Accessibility_Checker
+ */
+
+namespace EDAC\Admin;
+
+/**
+ * Class for inserting rule data into the database
+ *
+ * @since 2.0.0
+ */
+class Insert_Rule_Data {
+
+	/**
+	 * Insert rule data into database
+	 *
+	 * @since 2.0.0
+	 *
+	 * @param object $post     The post object.
+	 * @param string $rule     The rule.
+	 * @param string $ruletype The rule type.
+	 * @param string $rule_obj The object.
+	 *
+	 * @return void|int
```

Comment:
Update this to:
` @return void|int The ID of the newly inserted record, or void if no new record is inserted.`